src/makeCalculator.js

```diff
@@ -4,7 +4,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {};
+
+  calculator.result = 0;
+
+  calculator.operate = (callback, number) => {
+    calculator.result = callback(number);
```

Comment:
According to the task description, each `operate` call should take a callback and a number and set the appropriate value to the `result` property. But in your implementation, the `operate` method assigns the result of the callback invocation to the `result` property. This is not correct because the callback functions (add, subtract, multiply, divide) should modify the `result` property directly. So, you should change your callback functions to modify the `result` property directly and the `operate` method should just call the callback.